### ISSUE TITLE
1.3.3+ fails to install on Ubuntu

### ISSUE THREAD

**Issue by warnellg** [moses-palmer/pynput#31]
System: Ubuntu 14.04 / Python 2.7

Command to reproduce: `pip install pynput`

Error:
```
Collecting pynput
Using cached pynput-1.3.4-py2.py3-none-any.whl
Requirement already satisfied: six in /home/warnellg/anaconda2/lib/python2.7/site-packages (from pynput)
Collecting pyobjc-framework-Quartz>=3.0 (from pynput)
Using cached pyobjc-framework-Quartz-3.2.1.tar.gz
Complete output from command python setup.py egg_info:
Traceback (most recent call last):
File "<string>", line 1, in <module>
File "/tmp/pip-build-p6tKpe/pyobjc-framework-Quartz/setup.py", line 70, in <module>
[ 'Modules/_CVPixelBuffer.m' ]),
File "pyobjc_setup.py", line 375, in Extension
os_level = get_os_level()
File "pyobjc_setup.py", line 189, in get_os_level
pl = plistlib.readPlist('/System/Library/CoreServices/SystemVersion.plist')
File "/home/warnellg/anaconda2/lib/python2.7/plistlib.py", line 75, in readPlist
pathOrFile = open(pathOrFile)
IOError: [Errno 2] No such file or directory: '/System/Library/CoreServices/SystemVersion.plist'

----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-p6tKpe/pyobjc-framework-Quartz/
```

Comments: Perhaps related to commit `4c0eb7e84c962059f6d53ef7aa202c5d6f00d330` ("merge branch fixup-dependencies'"), since `pip` now tries to install `pyobjc` on Ubuntu 14.04. Rolling back to 1.3.2 resolves this issue (ie, `pip install pynput==1.3.2` succeeds).

**Comment by moses-palmer** [comment]
That must indeed be the case.

The reason, I think, is that *setuptools* on Ubuntu 14.04 does not support platform specifiers. Can you try installing inside of a *virtualenv* with an updated version of *setuptools*, and also with the Python 3 version of *pip*?

**Comment by warnellg** [comment]
Sure, but I just messed around with `virtualenv` for a bit, and I'm still fairly confused as to how to do what you've requested.

Can you provide some instructions?

**Comment by moses-palmer** [comment]
I tested installing using pip, and regardless of the version of pip and
setuptools installed, I got the same error. It seems that the format of
dependencies added in the commit you mentioned only works when installing
using setup.py.

I will have to revert the commit and make sure to somehow make it obvious
that you need to install python-xlib when running under X.

Thank you for your report.

**Comment by moses-palmer** [comment]
This has been fixed with version *1.3.5*.

The reason for this issue and [issue 30](https://github.com/moses-palmer/pynput/issues/30) was a [platform specifier](https://github.com/moses-palmer/pynput/commit/90ae3d377d5bc165edfdba9332d7551c7a3bacb1#diff-2eeaed663bd0d25b7e608891384b7298L33) for *python-xlib* incompatible with *Python 2*. The new release ensures that this dependency is added for legacy Python as well.
